src/app/constants.ts

```diff
@@ -84,3 +84,255 @@ export const AMOUNT_OF_RETRIEVE_NOTIFICATIONS_PER_CALL = 10
+  Togo: 'TG',
+  Philippines: 'PH',
+  Uzbekistan: 'UZ',
+  'Myanmar [Burma]': 'MM',
+  'British Virgin Islands': 'VG',
+  Zimbabwe: 'ZW',
+  'Congo - Kinshasa': 'CD',
+  'British Indian Ocean Territory': 'IO',
+  Montenegro: 'ME',
+  Indonesia: 'ID',
+  Dominica: 'DM',
+  Benin: 'BJ',
+  Angola: 'AO',
+  Sudan: 'SD',
+  'Congo - Brazzaville': 'CG',
+  Portugal: 'PT',
+  'New Caledonia': 'NC',
+  'North Korea': 'KP',
+  Grenada: 'GD',
+  Greece: 'GR',
+  'Cayman Islands': 'KY',
+  'Sint Maarten (Dutch Part)': 'SX',
+  Mongolia: 'MN',
+  Latvia: 'LV',
+  Morocco: 'MA',
+  Iran: 'IR',
+  'U.S. Minor Outlying Islands': 'UM',
+  Guyana: 'GY',
+  Guatemala: 'GT',
+  Iraq: 'IQ',
+  Chile: 'CL',
+  Nepal: 'NP',
+  'Heard Island and McDonald Islands': 'HM',
+  'Isle of Man': 'IM',
+  Ukraine: 'UA',
+  Tanzania: 'TZ',
+  Ghana: 'GH',
+  Curaçao: 'CW',
+  Anguilla: 'AI',
+  India: 'IN',
+  Canada: 'CA',
+  Maldives: 'MV',
+  Turkey: 'TR',
+  Belgium: 'BE',
+  Taiwan: 'TW',
+  'Trinidad and Tobago': 'TT',
+  'South Africa': 'ZA',
+  Bermuda: 'BM',
+  'Central African Republic': 'CF',
+  Jamaica: 'JM',
+  Turkmenistan: 'TM',
+  Peru: 'PE',
+  Germany: 'DE',
+  Tokelau: 'TK',
+  Fiji: 'FJ',
+  'United States': 'US',
+  Guinea: 'GN',
+  Somalia: 'SO',
+  Chad: 'TD',
+  'Pitcairn Islands': 'PN',
+  Thailand: 'TH',
+  Kiribati: 'KI',
+  'Equatorial Guinea': 'GQ',
+  'Costa Rica': 'CR',
+  Vietnam: 'VN',
+  'São Tomé and Príncipe': 'ST',
+  Nigeria: 'NG',
+  Kuwait: 'KW',
+  Croatia: 'HR',
+  Uruguay: 'UY',
+  'Sri Lanka': 'LK',
+  'Cook Islands': 'CK',
+  'Timor-Leste': 'TL',
+  'United Kingdom': 'GB',
+  Switzerland: 'CH',
+  Samoa: 'WS',
+  Spain: 'ES',
+  Venezuela: 'VE',
+  Liberia: 'LR',
+  'Burkina Faso': 'BF',
+  Swaziland: 'SZ',
+  'Saint Barthélemy': 'BL',
+  Palau: 'PW',
+  Estonia: 'EE',
+  'Wallis and Futuna': 'WF',
+  Niue: 'NU',
+  'South Korea': 'KR',
+  Austria: 'AT',
+  Mozambique: 'MZ',
+  'El Salvador': 'SV',
+  Monaco: 'MC',
+  Guam: 'GU',
+  Lesotho: 'LS',
+  Tonga: 'TO',
+  'Western Sahara': 'EH',
+  'South Sudan': 'SS',
+  Réunion: 'RE',
+  Hungary: 'HU',
+  Japan: 'JP',
+  Belarus: 'BY',
+  Mauritius: 'MU',
+  'Bouvet Island': 'BV',
+  'Norfolk Island': 'NF',
+  Albania: 'AL',
+  'New Zealand': 'NZ',
+  Senegal: 'SN',
+  Ethiopia: 'ET',
+  Egypt: 'EG',
+  'Sierra Leone': 'SL',
+  Bolivia: 'BO',
+  Malta: 'MT',
+  'Saudi Arabia': 'SA',
+  'Cape Verde': 'CV',
+  Pakistan: 'PK',
+  Gambia: 'GM',
+  Qatar: 'QA',
+  Ireland: 'IE',
+  'U.S. Virgin Islands': 'VI',
+  Slovakia: 'SK',
+  Serbia: 'RS',
+  Lithuania: 'LT',
+  France: 'FR',
+  'Bosnia and Herzegovina': 'BA',
+  Niger: 'NE',
+  Rwanda: 'RW',
+  'French Southern Territories': 'TF',
+  Bangladesh: 'BD',
+  Nicaragua: 'NI',
+  Barbados: 'BB',
+  Norway: 'NO',
+  'Saint Vincent and the Grenadines': 'VC',
+  Botswana: 'BW',
+  'Dominican Republic': 'DO',
+  Denmark: 'DK',
+  Uganda: 'UG',
+  Mexico: 'MX',
+  Suriname: 'SR',
+  Micronesia: 'FM',
+  'Saint Helena': 'SH',
+  Greenland: 'GL',
```

Comment:
**OPTIONAL. I think it will better to store this in a file called `contries.json` and called with a service because it is adding almost 300 lines to `constants.ts`

Reply:
Will do it, that's a great idea